### PR TITLE
Add internal LB resources

### PR DESCRIPTION
**Pull request by bpradipt**
Signed-off-by: Pradipta Banerjee <pradipta.banerjee@gmail.com>

### CODE REVIEW

File: data/data/powervs/loadbalancer/alb.tf [hunk 1]
```diff
@@ -10,6 +10,15 @@ resource "ibm_is_lb" "load_balancer" {
...
   security_groups = [ibm_is_security_group.ocp_security_group.id]
   tags            = [var.cluster_id]
+  type            = "public"
+}
+
+resource "ibm_is_lb" "load_balancer_int" {
+  name            = "${var.cluster_id}-loadbalancer-int"
+  subnets         = [var.vpc_subnet_id]
+  security_groups = [ibm_is_security_group.ocp_security_group.id]
+  tags            = [var.cluster_id]
```
Comment (clnperez) [inline data/data/powervs/loadbalancer/alb.tf]: the other providers put more than just the cluster_id https://github.com/openshift/installer/blob/65c320503d99a3eddffca3c35d74722b4f7c7ef1/data/data/aws/vpc/master-elb.tf#L10

so something like cluster_id-lbint

**Review by clnperez — CHANGES_REQUESTED** [review]

File: data/data/powervs/loadbalancer/alb.tf (hunk 1 above)
Comment (mkumatag) [reply data/data/powervs/loadbalancer/alb.tf]: can we have both as of now, will use whatever feasible while dealing with these resources.

Comment (bpradipt) [reply data/data/powervs/loadbalancer/alb.tf]: Done

**Review by mkumatag — APPROVED** [review]
LGTM

**Pull request merged**